packages/backend/src/tests/telemetry.ts

```diff
@@ -36,22 +31,14 @@ export class MockRatesService implements RatesService {
 export class MockTelemetryService implements TelemetryService {
   public aseRatesService = new MockRatesService()
   public internalRatesService = new MockRatesService()
-  public getOrCreateMetric(
-    _name: string,
-    _options?: MetricOptions | undefined
-  ): Counter<Attributes> {
-    return mockCounter
-  }
-  public getOrCreateHistogramMetric(
-    _name: string,
-    _options?: MetricOptions | undefined
-  ): Histogram<Attributes> {
-    return mockHistogram
-  }
+
+  incrementCounter(): void {}
+  async incrementCounterWithAmount(): Promise<void> {}
+  recordHistogram(): void {}
```

Comment:
we can also get rid of the `getBaseAssetCode` & `getBaseAssetCode` here as well?

Reply:
yup, removed